src/coreclr/nativeaot/Runtime/eventpipe/ep-rt-aot.cpp

```diff
@@ -502,4 +509,77 @@ ep_rt_aot_volatile_store_ptr_without_barrier (
     VolatileStoreWithoutBarrier<void *> ((void **)ptr, value);
 }
 
+void ep_rt_aot_init (void)
+{
+    extern ep_rt_lock_handle_t _ep_rt_aot_config_lock_handle;
+    extern CrstStatic _ep_rt_aot_config_lock;
+
+    _ep_rt_aot_config_lock_handle.lock = &_ep_rt_aot_config_lock;
+    _ep_rt_aot_config_lock_handle.lock->InitNoThrow (CrstType::CrstEventPipe);
```

Comment:
`CrstType` is unused in NativeAOT, but good for accounting where all the locks are. I think this should use a separate type, something like `CrstType::CrstEventPipeConfig`